doc/libpmem2/pmem2_vm_reservation_map_find.3.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEM2_VM_RESERVATION_MAP_FIND, 3)
+collection: libpmem2
+header: PMDK
+date: pmem2 API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2020, Intel Corporation)
+
+[comment]: <> (pmem2_vm_reservation_map_find.3 -- man page for libpmem2 pmem2_vm_reservation_map_find operation)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmem2_vm_reservation_map_find**() - finds the mapping stored in the virtual
+memory reservation
+
+# SYNOPSIS #
+
+```c
+#include <libpmem2.h>
+
+struct pmem2_vm_reservation;
+struct pmem2_map *pmem2_vm_reservation_map_find(
```

Comment:
change this function to return style to the one we use everywhere else: `int func(..., out *outp);`